### PR TITLE
Adding Baggage-Exo-ICA-Orchestration-Zones to Glossary 

### PR DESCRIPTION
**Pull request by mudassir-agoric**
Added entries for 

- Baggage
- Exo
- Interchain Accounts (ICA)
- Orchestration
- Zones

to glossary. Most of the text added was pulled from corresponding readme files.

### CODE REVIEW

**Comment by mudassir-agoric** [comment]
@dckc I am not confident with these concepts at all so please review (more) carefully. Thanks.

File: main/glossary/index.md [hunk 1]
```diff
@@ -620,3 +647,10 @@ patterns into reusable helpers. See the [Zoe Helpers API](/reference/zoe-api/zoe
 ## Zoe Service
 
 A set of API methods for deploying and working with smart contracts. See [Zoe Service API](/reference/zoe-api/zoe).
+
+## Zones
```
Comment (dckc) [inline main/glossary/index.md:651]:
```suggestion
## Zone
```

**Review by dckc — APPROVED** [review]
some suggestions included

File: main/glossary/index.md [hunk 2]
```diff
@@ -272,6 +276,16 @@ See [`E()`](#e) above.
 An object specifying how an [offer](#offer) can be cancelled, such as on demand or by a deadline.
 For details, see [`E(zoe).offer(...)`](/reference/zoe-api/zoe#proposals).
 
+## Exo
+
+An Exo object is an exposed Remotable object with methods (aka a `Far` object) which is
```
[the remark this answers is not in the document]
Comment (mudassir-agoric) [reply main/glossary/index.md:281]: Fixed.

File: main/glossary/index.md [hunk 3]
```diff
@@ -323,6 +337,10 @@ Guide](https://github.com/endojs/endo/blob/HEAD/packages/ses/docs/guide.md) for
 The Inter-Blockchain Communication protocol, used by blockchains to communicate with each other.
 For more details, see [What developers need to know about inter-blockchain communication](https://www.computerweekly.com/blog/Open-Source-Insider/What-developers-need-to-know-about-inter-blockchain-communication).
 
+## Intercahin Account (ICA)
+
+Interchain accounts on Agoric enable an Agoric smart contract to control an account on another blockchain within the Cosmos ecosystem facilitated by [Agoric Orchestration API](#orchestration). This feature leverages the [Inter-Blockchain Communication (IBC)](#ibc) protocol to facilitate interactions and transactions across different blockchains seamlessly.
```
[the remark this answers is not in the document]
Comment (mudassir-agoric) [reply main/glossary/index.md:342]: Fixed.

File: main/glossary/index.md [hunk 4]
```diff
@@ -463,6 +481,15 @@ the user either gets what they said they wanted, or gets back what they original
 escrowed (a refund). One reason this is possible is if a [proposal](#proposal) doesn't match what the contract expects to do, it
 can immediately cause the [seat](#seat) to exit, getting back the amount it offered.
 
+## Orchestration 
+
+Orchestration API is a tool to help developers build seamless applications out of disparate interoperable chains and services.
```
[the remark this answers is not in the document]
Comment (mudassir-agoric) [reply main/glossary/index.md:486]: Fixed.

**Pull request merged**